src/connect.js

```diff
@@ -20,7 +20,12 @@
  * @return {Function}
  */
 function connect(params) {
-  // write code here
+  const baseParams = {};
+
+  return (extraParams) => {
```

Comment:
Good, but you can write simply 
```suggestion
  return (extraParams) => { ...params, ... extraParams };
```